packages/shylu/shylu_dd/frosch/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 TRIBITS_SUBPACKAGE(FROSch)
 
-IF (${PROJECT_NAME}_ENABLE_ShyLU_DDFROSch AND NOT (HAVE_XPETRA_EPETRA AND HAVE_AMESOS2_EPETRA))
+IF (${PROJECT_NAME}_ENABLE_ShyLU_DDFROSch AND NOT (Xpetra_ENABLE_Epetra AND Amesos2_ENABLE_Epetra))
   SET(HAVE_SHYLU_DDFROSCH_EPETRA OFF)
```

Comment:
@mhoemmen Can you clarify what should be printed? This isn't intended to be a semantic change - I was trying to make it match the convention to use ``Xyz_ENABLE_Abc`` instead of ``HAVE_XYZ_ABC`` in CMake logic.